### PR TITLE
Started a readme and added some usage helpers

### PR DESCRIPTION
**Pull request by zack-erisyon**
Started a README, still a long way to go on it.

Added more clear usage strings in the common CLI mode where you have no idea what the args are. I just fixed them as they came along, it probably is an incomplete list of each mode.

Added a dye_seqs.tsv example file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## dye_seqs.tsv
+
+Tab delimited file.
+
+First two lines are special standalone ints that represent: n_channels and n_dye_seqs
+
+Followed by `n_dye_seqs` lines each of which is tab delimited with three terms:
+    * a "dye_string"
+    * the number of peptides that generated this dye_seq. TODO: Verify
```
Comment (mbsmith93) [inline README.md:47]: Verified. Remove TODO: Verify

File: README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    * the id of this dye_seq. TODO: Verify
```
Comment (mbsmith93) [inline README.md:48]: Verified. Remove TODO: Verify

Note for this though; when I produce the dye seq file from peptides, I assign an id to the dye seq from one of the peptides. The c++ code doesn't know how to read peptide data though so may not be super relevant, just helps with analysis to be able to guess at a particular peptide.

File: .gitignore [hunk 3]
```diff
@@ -2,3 +2,5 @@
 cc_code/bin/
 cc_code/build/
 python/**/__pycache__
+.idea/
```
Comment (mbsmith93) [inline .gitignore:5]: What's this? Just for my understanding, thanks

**Review by mbsmith93 — CHANGES_REQUESTED** [review]

**Comment by mbsmith93** [comment]
Thanks for this Zack, I appreciate it. Definitely hit me up with any questions; sorry about the umm... state of this code. I've not been the best with code cleanliness. If it helps I'll tell you you've at least gotten through the most horrific part, I think. The code in the src/io/ folder is awful. I was going to make it nicer, but then I decided "whatever it works it's fine." The src/hmm/step/ folder can get a little hairy as well, but the rest should just be your every-day standard c++ train-wreck and not have an extra dose of crazy like src/io/ or src/hmm/step/

File: .gitignore (hunk 3 above)
Comment (zack-erisyon) [reply .gitignore:5]: This is just ignoring the config files from my IDE

**Comment by zack-erisyon** [comment]
I addressed your comments. Safe to merge now.

**Review by mbsmith93 — APPROVED** [review]

**Pull request merged**